js/adapt-assessmentArticleModel.js

```diff
@@ -77,12 +77,31 @@ define([
       // save original children
       this._originalChildModels = this.getChildren().models;
 
+      this.setupCurrentQuestionComponents();
+
       this._setAssessmentOwnershipOnChildrenModels();
 
       // ensure the _questions attribute is set up (see https://github.com/adaptlearning/adapt_framework/issues/2971)
       this._updateQuestionsState();
     },
 
+    setupCurrentQuestionComponents: function() {
+      this._currentQuestionComponents = this.findDescendantModels('components', { where: { _isQuestionType: true } });
+
+      var assessmentConfig = this.getConfig();
+      this._currentQuestionComponents.forEach((component) => {
+        component.set({
+          _canShowFeedback: assessmentConfig._questions._canShowFeedback,
+          _canShowMarking: assessmentConfig._questions._canShowMarking,
+          _canShowModelAnswer: assessmentConfig._questions._canShowModelAnswer
+        });
+      });
```

Comment:
```suggestion
      const assessmentQuestionsConfig = this.getConfig()._questions;
      this._getAllQuestionComponents().forEach(component => {
        component.set({
          _canShowFeedback: assessmentQuestionsConfig._canShowFeedback,
          _canShowMarking: assessmentQuestionsConfig._canShowMarking,
          _canShowModelAnswer: assessmentQuestionsConfig._canShowModelAnswer
        });
      });
```

No need to store the array on the model instance.